capability/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2013 Suryandaru Triandana <syndtr@gmail.com>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
```

Comment:
I'm curious what the right approach is for this license file; currently this means that `golang.org/moby/sys/capability` will be licensed as MIT, but the LICENSE at the top of the repository is Apache2.


I know when vendoring other modules from this repository, `go mod` copies the LICENSE from the top-level, so I wonder what it does for this module (will it copy _both_? will it only use the MIT license?).

Reading these two StackOverflow answers;

- https://opensource.stackexchange.com/a/7391
- https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project

---

- At least the original license and copyright must be retained (see below)
- Possibly as a secondary file (?)
- Possibly included in a NOTICE file?

Reply:
I am now testing this PR in https://github.com/opencontainers/runc/pull/4358, and from what I see, `go mod vendor` uses top-level LICENSE file for e.g. moby/sys/mount and other packages, and it uses directory-level LICENSE for `moby/sys/capability`. In other words, it works as it should.

I don't see any problem with this layout -- a LICENSE file in a subdirectory overwrites the upper level LICENSE. If they are compatible (as in our case), all is fine and dandy.

Disclaimer: IANAL.

Reply:
Yeah, I'm somewhat looking from a "module" perspective, because it may become somewhat ambiguous; for the GitHub repo, the license on the repo-level is Apache 2, with a subdirectory having the MIT license.

But from a module perspective, normally it would copy that license; in this case it only has the MIT license. And perhaps that's OK (so, the `capabilities` module having a different license than the other modules), but was wondering if we wanted to re-license the new module to use Apache 2.

Reply:
☝️ s/MIT/BSD 2-clause/ (I was confused)

Reply:
> golang.org/moby/sys/capability will be licensed as MIT

For the sake of those reading this thread. This is actually a BSD 2-clause license, not the MIT license.
See https://opensource.org/license/bsd-2-clause